File: api/api/Models/fst/PersonnelinBranch.cs
using api.Models.Person;

namespace api.Models.fst
{
    public class PersonnelinBranch
    {
        public int ID { get; set; }
        public int PersonnelId { get; set; }
        public Personnel Personnel { get; set; }

        public int BranchId { get; set; }
        public Branch Branch { get; set; }

        public int PositionId { get; set; }
        public Position Position { get; set; }

        public int IsUsed { get; set; }
    }
}

File: fst.kru/src/models/PersonnelinBranch.ts
import { Branch } from "./Branch";
import { personnel } from "./Personnel";
import { Position } from "./Position";

export interface PersonnelinBranch {
  id: number;
  personnelId: number;
  personnel: personnel;
  branchId: number;
  branch: Branch;
  positionId: number;
  position: Position;
  isUsed: number;
  hidden: boolean;
}
